plugins/mail/nullmailer_queue

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
```

Comment:
Below there is only a single occurrence of a bash-only feature: `&>` (which can be easily replaced). Maybe you want to turn this into a `/bin/sh` plugin?

Reply:
Apparantly, I used two features, since I also used `[[ … ]]` which is also a bash-only feature. In general, it is more flexible and safe to use than `[ … ]` as well as being a shell syntax feature rather than an external command, though in bash and zsh, it is a builtin. Anyway, since I didn't use those features I changed it to `[ … ]`.

If you'd like, you can read more about it here: http://mywiki.wooledge.org/BashFAQ/031